<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Симка и нолик</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resursi.qrc">
    <normaloff>:/images/iconka.ico</normaloff>:/images/iconka.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/bg.png);
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>421</width>
      <height>431</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>450</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/bgbutton.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/bgbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/bgbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/bgbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/bgbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/bgbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/bgbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/bgbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/bgbutton.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>34</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/bgbutton.png);
border-radius:19px;
color: white;</string>
    </property>
    <property name="text">
     <string>Об игре</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HoloLens MDL2 Assets</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: white;</string>
    </property>
    <property name="text">
     <string>ТЫ ВИРТУОЗ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Medium</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/bgbutton.png);
border-radius:19px;
color: white;</string>
    </property>
    <property name="text">
     <string>Играть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/bgbutton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>971</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/images/bandag.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>bg</zorder>
   <zorder>label_2</zorder>
   <zorder>tabWidget</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_11</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resursi.qrc"/>
 </resources>
 <connections/>
</ui>
